src/sparse_voxels.jl

```diff
@@ -79,24 +76,25 @@ function SparseVoxelGrid{T <: AbstractVector}(points::Vector{T}, voxel_size)
     return SparseVoxelGrid(voxel_size, voxel_info, point_indices)
 end
 
-# Convert matrix to Vector{SVector}
-function SparseVoxelGrid{T <: Number}(points::Matrix{T}, voxel_size)
-    ndim = size(points, 1)
```

Comment:
Maybe check if `size(points,1)` matches `voxel_size`?  Also seems a bit unnecessary that `voxel_size` is strictly an `SVector`?